<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
background-color: rgb(61,61,61);
border-radius: 15px;
color: rgb(255,255,255);
font: 77 10pt&quot;Arial Black&quot;;

}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgb(61,61,61);
border-radius: 15px;
color: rgb(255,255,255);
font: 77 10pt&quot;Arial Black&quot;;

}

QPushButton:hover {
background-color:rgb(0,206,151);
border-radius: 15px;
color: rgb(0,0,0);
font: 77 10pt &quot;Arial Black&quot;;
}
   </string>
   </property>
   <widget class="QPushButton" name="btnLed01">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>LED 01</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLed02">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(61,61,61);
border-radius: 15px;
color: rgb(255,255,255);
font: 77 10pt&quot;Arial Black&quot;;

}

QPushButton:hover {
background-color:rgb(0,206,151);
border-radius: 15px;
color: rgb(0,0,0);
font: 77 10pt &quot;Arial Black&quot;;
}
   </string>
    </property>
    <property name="text">
     <string>LED 02</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLed04">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>LED 04</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLed03">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>LED 03</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLed09">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>150</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>LED 09</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLed08">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(61,61,61);
border-radius: 15px;
color: rgb(255,255,255);
font: 77 10pt&quot;Arial Black&quot;;

}

QPushButton:hover {
background-color:rgb(0,206,151);
border-radius: 15px;
color: rgb(0,0,0);
font: 77 10pt &quot;Arial Black&quot;;
}
   </string>
    </property>
    <property name="text">
     <string>LED 08</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLed07">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>LED 07</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLed06">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>80</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>LED 06</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLed05">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>LED 05</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLed12">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>210</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>LED 12</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLed11">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>LED 11</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLed10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>LED 10</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>511</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {

    background-color: rgb(170, 255, 255);

}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>btnLed01</zorder>
   <zorder>btnLed02</zorder>
   <zorder>btnLed04</zorder>
   <zorder>btnLed03</zorder>
   <zorder>btnLed09</zorder>
   <zorder>btnLed08</zorder>
   <zorder>btnLed07</zorder>
   <zorder>btnLed06</zorder>
   <zorder>btnLed05</zorder>
   <zorder>btnLed12</zorder>
   <zorder>btnLed11</zorder>
   <zorder>btnLed10</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
